src/test/java/com/ge/predix/uaa/token/lib/ZacTokenServiceTest.java

```diff
@@ -215,11 +215,11 @@ private void assertAuthentication(final OAuth2Authentication authentication, fin
     public void testGetServiceHeaders() {
```

Comment:
pls add a new test which tries to use ZacTokenService with neither headers or subdomain defined, it should throw IllegalStateException or similar